website/content/docs/nia/enterprise/index.mdx

```diff
@@ -7,8 +7,21 @@ description: >-
 
 # Consul-Terraform-Sync Enterprise
 
-Consul-Terraform-Sync Enterprise features address organization complexities of collaboration, operations, scale, and governance. Please see the sidebar navigation on the left to view all enterprise features.
+Consul-Terraform-Sync (CTS) Enterprise is available with [Consul Enterprise](https://www.hashicorp.com/products/consul) and requires a Consul [license](/docs/nia/enterprise/license) to be applied.
 
-Enterprise features require a Consul [license](/docs/nia/enterprise/license) to be applied.
+Enterprise features of CTS address organization complexities of collaboration, operations, scale, and governance. CTS Enterprise supports an official integration with [Terraform Cloud](https://www.terraform.io/cloud) to extend insight into dynamic updates of your network infrastructure.
 
-These features are part of Consul-Terraform-Sync Enterprise which is part of [Consul Enterprise](https://www.hashicorp.com/products/consul).
+| Features | Open Source | Enterprise |
+|----------|-------------|------------|
+| | Self-Managed | Self-Managed |
+| Consul Namespace | Default namespace | Filter task triggers by namespace |
+| Network Automation | Terraform OSS | Terraform OSS or Terraform Cloud |
```

Comment:
The entry for Enterprise should also include Terraform Enterprise

Reply:
Support for TFE was rolled out first, and TFC later, it made sense to have the distinction. Now that they are both supported, could we move forward with only using one name unless we're referencing distinct features not available in both? This would follow the naming usage across the terraform.io documentation.

For example, the TFC with audit trails and TFE with audit logging

Reply:
There is clarification that TFE is supported in 3 other places I think should remain -- the network drives page, configuration, and compatibility chart

Reply:
Although Terraform team is using TFC to position both cloud and self-managed(TFE), the reality is that our users/customers are still not clear that TFC positioning now includes TFE. Including TFE will mitigate any such confusion

Reply:
That's true, alright i'll add TFE to help bridge that gap